gravpm.c

```diff
@@ -276,11 +276,16 @@ void powerspectrum_compute(const int64_t k2, const int kpos[3], pfft_complex * c
     /* Measure power spectrum: we don't want the zero mode.
      * Some modes with k_z = 0 or N/2 have weight 1, the rest have weight 2.
      * This is because of the symmetry of the real fft. */
-    int kint = floor(sqrt(k2));
-    if(kint >= 0 && kint < PowerSpectrum.size) {
+    if(k2 > 0) {
```

Comment:
It is because the power spectrum is normally plotted in log units, and normally you care more about what happens at low k than high k. So log scaling is a cheap way of placing more bins at low k. Practically this makes, eg, the BAO peak look like a rounded bump instead of a sharp peak. Integer binning would work too - one can easily change it back later though.